node/auth/authenticator_test.go

```diff
@@ -516,8 +325,12 @@ func TestKeyCacheSize(t *testing.T) {
 	require.NoError(t, err)
 	request.Signature = signature
 
-	err = authenticator.AuthenticateStoreChunksRequest(context.Background(), "0", request, start)
+	hash, err = authenticator.AuthenticateStoreChunksRequest(context.Background(), request, start)
+	require.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
duplicate no error check

Reply:
I just want to be REALLY sure there wasn't an error. Perhaps I should spin here in a for loop just in case. (fixed)